lib/Github/Api/Environment.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Github\Api;
+
+/**
+ * Listing, creating and updating environments.
+ *
+ * @link https://docs.github.com/en/rest/deployments/environments?apiVersion=2022-11-28#
+ */
+class Environment extends AbstractApi
+{
+    use AcceptHeaderTrait;
```

Comment:
This api doesn't require a custom beta accept header, so the trait usage can be removed

Reply:
Hi @acrobat thank you, is now removed.